examples/llms/structured.ts

```diff
@@ -2,9 +2,11 @@ import "dotenv/config.js";
 import { z } from "zod";
 import { BaseMessage, Role } from "bee-agent-framework/llms/primitives/message";
 import { OllamaChatLLM } from "bee-agent-framework/adapters/ollama/chat";
+import { JsonSchemaGenerator } from "bee-agent-framework/generators/jsonSchemaGenerator";
```

Comment:
I would change the module name from `jsonSchemaGenerator` to `json` for more neat imports.